lib/menu.rb

```diff
@@ -0,0 +1,44 @@
+class Menu
+  attr_accessor :list_of_dishes
+
+  def initialize
+    @list_of_dishes = []
+  end
+
+  def add_to_menu(dish)
+    @list_of_dishes.push(dish.itemise)
+  end
+
+  def access_menu
+    @list_of_dishes = DEFAULT_MENU
+  end
+
+  def view_menu
+    access_menu if @list_of_dishes.empty?
+    banner_display
```

Comment:
If we want to strictly respect the SRP, we ideally don't want to have things that deal with "presentation" (printing/display, etc) in the same class or component that deals with data. So maybe it'd be good to move the methods `menu_display` and `banner_display` into their own class (e.g `MenuDisplay`), and this class instance could be dependency-injected into `Menu` (then we could pass the list of dishes in arguments to those methods, if needed).